files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -718,3 +718,22 @@
     name: "nullmailer"
     state: "stopped"
   when: not ansible_is_chroot
+
+#
+# Add the correct path to /etc/security/pam_env.conf and remove any invalid
+# paths from /etc/environment to ensure that non-existent paths are not
+# included in the global PATH.
+#
+- lineinfile:
+    path: /etc/security/pam_env.conf
+    state: present
+    regexp: '^\s*PATH\s+DEFAULT='
+    line: 'PATH DEFAULT=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin'
+
+#
+# This can be removed once the minimum upgrade version is 28.0.0.0.
```

Comment:
As the comment describes in `gradle.properties`, we should add a string of the form `MIN_UPGRADE_VERSION(28.0.0.0)` here so that we can search for it later.